recipes/pyscenarios/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: pyscenarios
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pyscenarios/pyscenarios-{{ version }}.tar.gz
+  sha256: 362d13c8e5d51bd9badf104bf5c3616e0d5b64f5e3bb9934aa23dd168c6b590d
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python>=3.5
+    - dask>=0.17.3
```

Comment:
this should probably only need python and pip at build time

Reply:
Nope. There is a resource file which is pre-calculated at build time, and it takes the whole library to do so.

Reply:
that's unfortunate